it/test/controllers/claimToAdjustPoa/AmendablePOAControllerISpec.scala

```diff
@@ -53,6 +57,64 @@ class AmendablePOAControllerISpec extends ComponentSpecBase {
           httpStatus(OK)
         )
       }
+      "User is authorised and there is an active session and journeyCompleted = false" in {
```

Comment:
when writing test, please make sure there is at least one scenario with amount containing the fractional part ie: £250.13 etc